.github/workflows/test-all-packages.yml

```diff
@@ -7,6 +7,11 @@ on:
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
+env:
+  AVA_USE_TAP: true
```

Comment:
Ava has a [CLI option for using tap](https://github.com/avajs/ava/blob/main/docs/05-command-line.md#tap-reporter), `--tap`. Let's not make a new one if we can avoid it.

If that's too much work then please name this something like `AGORIC_AVA_FORCE_TAP` and down in the patchfile point to the `--tap` docs and explain why we have an env var instead.

Reply:
this didn't propagate through both yarn and lerna appropriately unfortunately, forcing me to take this route

Reply:
Hmm. Maybe it needs an extra `--`? Not a blocker